infrastructure-components.yaml

```diff
@@ -0,0 +1,5464 @@
+apiVersion: v1
```

Comment:
so we need to have this `infrastructure-components.yaml` in the source tree, or do we just need it uploaded as an asset to the release?

if we need it in the source tree, then we need something (a CI job) to ensure it's always up-to-date.  If we don't need this yaml for the quickstart in the source tree itself, then i say we don't commit it and only upload it in it's generated form to the github release.

Reply:
It only needs to be in the release.

Reply:
Deleted and gitignored it.